### ISSUE TITLE
Missing basic description of how this works in the documentation/specification. How do I prevent losing tabs?

### ISSUE THREAD

**Issue by adam-kral** [Drive4ik/simple-tab-groups#947]
After reading the specs of this extension I wasn't sure if the tab groups with the tabs would be preserved after browser crash/closure. Therefore I needed to inspect it _manually_, myself. After installing the extension, I got even more unsure this would work, seeing the requirement to check the `reopen session` in Firefox settings. Finally, I played with closing/killing/quitting Firefox. And found out, that when I do cmd-Q on mac (quit) I lose all the tabs! (Killing Firefox restored the tabs – but! I have experience with Firefox of many years. And over the last say 3 years, at least 3 times it occurred to me that it _could not_ restore my tabs after crash/exiting and there was no coming back.)

The point of this extensions is you can avoid having ever-increasing count of windows/tabs. But if you cannot safely close the browser, that defies the purpose. I am not keen on building the tab groups for months and having a sword of Damocles above me, that I accidentally press cmd-Q or firefox crashes and session restoration wouldn't be possible and _everything is gone_.

So I have several _essential_ questions (I apologize for the tone): 

1) How do I prevent losing the tabs, as described above. 
2) If I can prevent it, why is it not enabled out-of-the-box? 
3) Why isn't it described on addons.mozilla, or here on github readme – what happens after closing window/exiting the browser. That should be one of the first things there. Or describe the shortcoming, if it is not possible to prevent losing the tabs. Certainly better than leaving people in blissful ignorance.

**Comment by adam-kral** [comment]
Ok, so I found out about the backup functionality. In the default setting, the user has up to a month to restore the backup before it gets overwritten. That is enough.

However, the backups in the day are always overwritten, so I think there's of little use to setting the backup interval to less than a day -> it may get overwritten (by the backup of clean empty browser after a crash) before you manage to restore from it.  

In other words, you can only be sure you can restore the yesterdays state, even if you set the backup interval to an hour or a minute. (So in a way, the ability to set minute/hourly backups can mislead the people in believing they could only lose minute/hour of work.) Actually, if you se it to a minute range, it will almost surely get overwritten by the clean browser backup before you even take the first step to restore them.
